src/main/java/net/earthcomputer/clientcommands/command/AutoPrefixCommand.java

```diff
@@ -0,0 +1,58 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.CommandDispatcher;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.commands.CommandBuildContext;
+import net.minecraft.network.chat.Component;
+
+import static com.mojang.brigadier.arguments.StringArgumentType.*;
+import static dev.xpple.clientarguments.arguments.CGameProfileArgument.*;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.*;
+
+public class AutoPrefixCommand {
+    private static String currentPrefix = "";
+
+    public static String getCurrentPrefix() {
+        return currentPrefix;
+    }
+
+    public static void register(CommandDispatcher<FabricClientCommandSource> dispatcher, CommandBuildContext context) {
+        dispatcher.register(literal("cautoprefix")
```

Comment:
a short-circuit branch to get the prefix would be nice

Reply:
just query lazy